docs/algorithms/training.rst

```diff
@@ -0,0 +1,56 @@
+===============================
+Training Information for Models
+===============================
+Models described below have a "version tag" described for them, which is
+intended to be a canonical identifier for that model instance.
+
+Object Detection Models
+=======================
+
+
+Tourniquet Model Stage2 with interstitial steps
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Version Tag: ``tourniquet_stage2_interstitial``
+
+
+Training data:
+    - BBN's M2_Tourniquet v0.52 videos: ['M2-131', 'M2-63', 'M2-71', 'tq_2', 'M2-11', 'tq_4', 'M2-54',
+            'M2-73', 'M2-62' 'M2-46', 'M2-60', 'M2-24', 'M2-19', 'M2-2', 'M2-74', 'M2-28', 'M2-135',
+            'M2-10', 'M2-61',  'M2-137', 'M2-53', 'M2-40', 'M2-66', 'M2-138', 'M2-124', 'M2-102', 'M2-32',
+            'M2-29','M2-41', 'M2-1' 'M2-127'   'M2-23', 'M2-17' 'M2-139', 'M2-55', 'M2-69' 'M2-68', 'M2-13',
+            'M2-133', 'M2-121', 'M2-14', 'M2-50', 'M2-44', 'M2-43', 'M2-35', 'M2-75', 'M2-37']
+    - BBN's M2 Lab Data (HoloLens videos):  ['tq_12', 'tq_8', 'tq_21', 'tq_23', 'tq_3', 'tq_20', 'tq_19',
+            'tq_14', 'tq_13', 'tq_10', 'tq_11', 'tq_5']
+
+
+Validation data:
+    - BBN's M2_Tourniquet v0.52 videos: ['M2-72', 'M2-58', 'M2-134', 'M2-105', 'M2-33', 'M2-9', 'M2-42',
+            'M2-30', 'M2-16', 'M2-128']
+    - BBN's M2 Lab Data (HoloLens videos): ['tq_18', 'tq_22']
```

Comment:
I don't see the value of this info being separated out over here as opposed to just being under the relevant models described in the `docs/algorithms/model_versions.rst` file.